src/Threads/Provider.php

```diff
@@ -82,7 +82,7 @@ protected function mapUserToObject(array $user)
             'nickname'     => Arr::get($user, 'username'),
             'name'         => null,
             'email'        => null,
-            'avatar'       => Arr::get($user, 'threads_profile_picture_url'),
+            'avatar'       => Arr::get($user, 'threads_profile_picture_url') ?? null,
```

Comment:
```suggestion
            'avatar'       => Arr::get($user, 'threads_profile_picture_url'),
```

No need to add the null coalesce

Reply:
hmm, oke thank you. i will fix soon.